build.gradle

```diff
@@ -78,7 +78,8 @@ buildscript {
       desugarJdk               : '1.0.9',
       signaturePad             : '1.3.1',
       viewpager2               : '1.0.0',
-      lint                     : '27.0.1'
+      lint                     : '27.0.1',
+      rootbeer                 : '5fef9f62f9'
```

Comment:
Are we using Jitpack version rather than maven?

Reply:
The library is not pushed to maven central, and the latest official release is months old. A bunch of fixes for false positives have gotten into `master` without an official release, so I opted to fetch the current `master` commit instead.